dotnetUno5/Dockerfile.debug

```diff
@@ -0,0 +1,111 @@
+# ARGUMENTS --------------------------------------------------------------------
+##
+# Board architecture
+##
+ARG IMAGE_ARCH=
+# For armv7 use:
```

Comment:
About this comment, does it makes sense remove it from all templates? As this is set automatically when running the extension and there is the validate arch step, the user should not hardcode this

Reply:
yeah I think that make sense